builders/integrations/oracles/api3.md

```diff
@@ -0,0 +1,365 @@
+[*To know more about how dAPIs work, click here*](https://docs.api3.org/explore/dapis/what-are-dapis.html)
+
+<!-- ### Types of dAPIs
+
+#### Self-funded dAPIs
+Self-funded dAPIs offer developers the opportunity to experience data feeds with
+minimal up-front commitment, providing a low-risk option prior to using a
+managed dAPIs.
+
+#### Managed dAPIs
+Managed dAPIs are sourced from multiple first-party oracles and aggregated using
+a median function. Compared to self-funded dAPIs, **managed dAPIs are monetized**,
+as API3 requires payment in USDC on Ethereum Mainnet to operate them. -->
+
+### Subscribing to self-funded dAPIs {: #self-funded-dapis}
+
+With self-funded dAPIs, you can fund the dAPI with your own funds. The amount of gas you supply will determine how long your dAPI will be available for use. If you run out of gas, you can fund the dAPI again to keep it available for use.
+
+#### Exploring and selecting your dAPI {: #select-dapi}
+
+The [API3 Market](https://market.api3.org/dapis) provides a list of all the dAPIs available across multiple chains including testnets. You can filter the list by chains and data providers. You can also search for a specific dAPI by name. Once selected you will land on the details page where you can find more information about the dAPI.
+
+#### Funding a sponsor wallet {: #fund-wallet}
+
+Once you have selected your dAPI, you can activate it by using the [API3 Market](https://market.api3.org/) to send DEV to the `sponsorWallet`. Make sure your:
+
+- Wallet is connected to the Market and is the same network as the dAPI you are funding.
+- Balance of the wallet should be greater than the amount you are sending to the `sponsorWallet`.
+
+> ![API3 Remix deploy](/images/builders/integrations/oracles/api3/SS1.png)
+
+To fund the dAPI, you need to click on the **Fund Gas** button. Depending upon if a proxy contract is already deployed, you will see a different UI.
+
+> ![API3 Remix deploy](/images/builders/integrations/oracles/api3/SS2.png)
+
+Use the gas estimator to select how much gas is needed by the dAPI. Click on **Send DEV** to send the entered amount to the sponsor wallet of the respective dAPI.
+
+> ![API3 Remix deploy](/images/builders/integrations/oracles/api3/SS3.png)
+
+Once the transaction is broadcasted & confirmed on the blockchain a transaction confirmation screen will appear.
+
+> ![API3 Remix deploy](/images/builders/integrations/oracles/api3/SS5.png)
+
+#### Deploying a proxy contract to access the dAPI {: #deploy-proxy}
+
+Smart contracts can interact and read values from contracts that are already deployed on the blockchain. By deploying a proxy contract via the API3 Market, a dApp can interact and read values from a dAPI like ETH/USD.
+
+!!! note
+    If a proxy is already deployed for a self-funded dAPI, the dApp can read the dAPI without having to deploy a proxy contract. They do this by using the address of the already deployed proxy contract which will be visible on the API3 Market.
```

Comment:
where exactly would the address of the already deployed proxy be?

Reply:
If a proxy is already deployed, the user just needs to make sure the dAPI is funded and can start using it. It would be on the dAPI page when the user clicks on the **Get Proxy** button.
![image](https://user-images.githubusercontent.com/17065256/233772829-cb8b3e21-0aa2-4b53-a572-55e5c0962427.png)